source/common/environment.js

```diff
@@ -24,7 +24,7 @@ const environment = Object.assign({
   TEST: 'test',
   PRODUCTION: 'production',
   NETWORK: process.env.NETWORK || 'development',
-  API: process.env.API || 'ada',
+  API: 'ada',
```

Comment:
@daniloprates I think we can remove this line as it is no longer used in the code.